### PR TITLE
Deploying v51 (and bumping version to v52) 

### PR DESCRIPTION
**Pull request by lundjordan**
This Pull Request contains the version bump and should be merged once deployment is successful.
Deployment coordinator for this release is: @rail  

# Deployment to staging

- Build1 -> https://tools.taskcluster.net/groups/Z1lS85k0T-mrWYZBhSIU9Q

## Verifications

- [x] docs
- [ ] codecoverage/backend
- [ ] codecoverage/bot
- [ ] codecoverage/crawler
- [ ] pulselistener
- [ ] releng-frontend
- [ ] mapper/api
- [x] notification/identity
- [x] notification/policy
- [ ] tokens/api
- [ ] tooltool/api
- [ ] treestatus/api
- [x] shipit/api
- [x] shipit/frontend
- [ ] staticanalysis/bot
- [ ] staticanalysis/frontend
- [ ] uplift/backend
- [ ] uplift/bot

# Deployment to production

- Build1 -> https://tools.taskcluster.net/groups/BZQ5o57mSmmkFHQNX3FqZw

## Verifications

- [x] docs
- [ ] codecoverage/backend
- [ ] codecoverage/bot
- [ ] codecoverage/crawler
- [x] pulselistener
- [ ] releng-frontend
- [ ] mapper/api
- [ ] notification/identity
- [ ] notification/policy
- [ ] tokens/api
- [ ] tooltool/api
- [ ] treestatus/api
- [x] shipit/api
- [x] shipit/frontend
- [x] staticanalysis/bot
- [x] staticanalysis/frontend
- [x] uplift/backend
- [x] uplift/bot

### CODE REVIEW

**Comment by jankeromnes** [comment]
Passing `staticanalysis/bot` (go for release), but mentioning two tiny issues for the record:

- 1 undetected defect in https://phabricator.services.mozilla.com/D11320
- 1 undetected defect in https://phabricator.services.mozilla.com/D11321

(Currently unable to debug more, because the dashboard has a cloudfront 403 error and says "No issues" e.g. in https://static-analysis.moz.tools/#/task/Zwpd67jTRLaahqynL0ycHg )

**Comment by marco-c** [comment]
> mentioning two tiny issues for the record

Could you file them as issues so we don't forget?

**Comment by garbas** [comment]
@lundjordan Looks like this is not bumping to 52. the PR should be in failing state until we will push base image 52 out.

@jankeromnes this PR is for next release you probably want to check off things [here](https://github.com/mozilla/release-services/pull/1672) (i did this already for you)

File: .taskcluster.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ tasks:
     priority: high
     payload:
       maxRunTime: 1800  # 30min
-      image: "mozillareleng/services:base-50"
+      image: "mozillareleng/services:base-51"
```
Comment (garbas) [inline .taskcluster.yml]:
```suggestion
      image: "mozillareleng/services:base-52"
```

File: .taskcluster.yml [hunk 2]
```diff
@@ -72,7 +72,7 @@ tasks:
...
```
Comment (garbas) [inline .taskcluster.yml]:
```suggestion
      image: "mozillareleng/services:base-52"
```

File: .taskcluster.yml [hunk 3]
```diff
@@ -112,7 +112,7 @@ tasks:
...
```
Comment (garbas) [inline .taskcluster.yml]:
```suggestion
      image: "mozillareleng/services:base-52"
```

File: .taskcluster.yml [hunk 4]
```diff
@@ -152,7 +152,7 @@ tasks:
...
```
Comment (garbas) [inline .taskcluster.yml]:
```suggestion
      image: "mozillareleng/services:base-52"
```

File: .taskcluster.yml [hunk 5]
```diff
@@ -192,7 +192,7 @@ tasks:
...
```
Comment (garbas) [inline .taskcluster.yml]:
```suggestion
      image: "mozillareleng/services:base-52"
```

File: VERSION [hunk 6]
```diff
@@ -1 +1 @@
-50
+51
```
Comment (garbas) [inline VERSION]:
```suggestion
52
```

**Comment by marco-c** [comment]
> @marco-c Could you update the prod secret with the following changes for `staticanalysis/bot`:

Done! clang-format was already in the analyzers list, so I've only added ALLOWED_PATHS.

**Comment by rail** [comment]
We had some issues with the token/api service, which was rolled back to v50.

**Comment by garbas** [comment]
@rail I just remembered that new PR with a bumped version needs to be created. Do you have time to do this or should I do it

**Pull request merged**